### DISCUSSION TITLE
I hate this update please give option to use the old one!

### DISCUSSION THREAD

**Discussion by DrAhmedAliFahmey** [community/community#54984] [Accessibility]
### Select Topic Area

Product Feedback

### Body

Give people chance to choose what they want to use  I'm sure in not the only one who want the old version back

**Comment by ghostinhershell** [comment]
Hi @DrAhmedAliFahmey, 

Thank so much for writing in and thanks so much for posting in the Community on Discussions. Can you be a bit more specific about the update that you're referring to here?
